packages/shared/ReactOwnerStackReset.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import ReactSharedInternals from 'shared/ReactSharedInternals';
+
+let resetOwnerStackIntervalId: mixed = null;
```

Comment:
each renderer has its own interval now. Should we just move this timeout ID into isomorphic React?